### ISSUE TITLE
Feature Suggestion - PDF Report

### ISSUE THREAD

**Issue by larix93** [JesseBonanno/IndeterminateBeam#30]
Currently, the report is output as an HTML but a pdf report would be a small and simple addition.

**Comment by JesseBonanno** [comment]
Agreed. I have previously had difficulty in generating a PDF. I will take another look to see if i can come up with a method to achieve this.

**Comment by JesseBonanno** [comment]
Unfortunely this is not a simple addition and is extremely complicated. If anyone wants to take a look these are the most useful resources that I found: 
- https://stackoverflow.com/questions/67166979/dash-extensions-problem-downloading-pdf-file-generated-as-a-byte-stream 
- https://gist.github.com/philfreo/44e2e26a65820497db234d0c66ed58ae
- https://plotly.com/python/v3/pdf-reports/

I did manage to get some pdfs generating however i could not get the graphs to show as html or as images.

I am currently working on a django implementation of the web app in which i could get pdfs to generate. This version is not currently hosted but can be forked and hosted privately. https://github.com/JesseBonanno/django-indeterminatebeam

I am leaving the issue open so that someone else may address it.

**Comment by JesseBonanno** [comment]
Is semi completed with commit #8fb4989
Have been able to include the diagrams in a report. Should be able to use the same approach to only export the graphs to pdf and do this for both indeterminate and determinate beams.
